#183838931 - Pre Register (Sign Up)

Sirjoseph94 opened the pull request:
#### What does this PR do?
Start the pre-register ticket

#### Description of Task to be completed?
Design the UI of the Pre-register ticket

#### What are the relevant pivotal tracker stories? 
https://www.pivotaltracker.com/story/show/183838931

Commit:
Refactor(input field): Create resuable input field

Previously, React-hook-form was used to handle form submission and validation. Now I changed it to vanilla React and Css to reduce the number of packages

"

Commit:
Refactor(codebase): css library

Removed MUI and replaced with TailwindCSS

Commit:
Chore(Setup): Integrate tailwind and husky

Configure tailwind css and install husky hook

Commit:
Feature(project): husky setup

install husky hook to check for syntax errors before pushing

Commit:
Chore(project): Resolve merge conflict

Resolve merge conflict

Commit:
Documentation(project): create README

Create detailed README

Commit:
Chore(README): Add badges to README

added dynamic badges to README

Commit:
House keeping

Commit:
Feature(Auth): Create pre register file structure

Create the Pre-register file structure

Chisonm reviewed (commented):
Please maintain lowercase for folder name

Chisonm reviewed (changes requested):
Create a seperate folder called "shared" and put all reuseable components inside

Sirjoseph94 commented:
> Create a seperate folder called "shared" and put all reuseable components inside

There is already a folder called components for reusable components, you want me to rename it?

Sirjoseph94 commented:
> Please maintain lowercase for folder name

Noted

Commit:
renamed all folder names to lowercase

Chisonm commented:
> > Create a seperate folder called "shared" and put all reuseable components inside
> 
> There is already a folder called components for reusable components, you want me to rename it?

create a folder called shared and put all reuseable components inside eg: components/shared/input

Commit:
feature(pre-register): starts pre register task

start pre register
[start #183838931]

Commit:
Feature(Pre-register): Pre-register page design

I designed the pre register page in accordance to figma design, there is still need for some tweaking
[Delivers #183838931]
